app/services/spree_avatax_official/transactions/create_service.rb

```diff
@@ -42,6 +46,20 @@ def send_request(order, transaction_type, options)
 
         client.create_transaction(create_transaction_model, options)
       end
+
+      def log_unusual_order(order, response)
+        Airbrake.notify('[AVATAX] Unusual order with total higher than 100k:', {
```

Comment:
Just double checking - when do do `Airbrake.notify` without an exception, will it also include a stack trace? I'm not sure how this works under the hood, but it would be great to capture it too :)